UserManagement/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM java:8
+FROM maven:alpine
+WORKDIR /app
+COPY . /app
+RUN mvn -v
+RUN mvn clean install -DskipTests
+EXPOSE 8181
+ADD ./target/UserManagement-0.0.1-SNAPSHOT.jar /jar/
```

Comment:
I think we can just create a maven goal to run the main class and then do something like `mvn exec:main` to run the application. also please check again if we need tomcat as a part of this image.

Reply:
I think we wouldn't need the tomcat service for this image. we just need the jar that can be created with maven. looking into that right now.